util/prefix_validate_test.go

```diff
@@ -0,0 +1,40 @@
+package util_test
```

Comment:
When we change the file name of the `prefix_validate.go` file, we need to think of renaming this file as well.